RELEASE-NOTES.md

```diff
@@ -1,7 +1,13 @@
 # About this release
 
-Changes:
-* Add [small localnet](docs/docker.md#localnet) docker-compose configuration for testing and development purposes
+[This release](https://github.com/aeternity/epoch/releases/tag/v0.7.0) is focused on smart contracts: it provides HTTP API for compiling and running smart contract and creating contract transactions.
+It also:
+* Refines the naming system by validating names and labels;
+* Reviews hashing in the blockchain, preferring BLAKE2b;
+* Expands the user API;
+* Adds a [small localnet](docs/docker.md#localnet) docker-compose configuration for testing and development purposes;
+* Refines the usage of the persisted database for the blockchain, optimizing memory usage;
+* Improves the stability of the testnet.
 
 This release introduces backward incompatible changes in the chain format.
```

Comment:
Is it due to switching to Blake2b? If yes, I would explicitly state that switching to the new hashing function introduces backward incompatible changes.